### PR TITLE
feat(tslint): connect tslint

### PR DESCRIPTION
**Pull request by mdmitriy**

### CODE REVIEW

File: demo/src/components/content-types/content-types.component.ts [hunk 1]
```diff
@@ -26,9 +25,9 @@ import { ContentfulService } from '../../../../src/services/contentful.service';
   `
 })
 export class ContentTypesComponent implements OnInit {
-  static RoutingName = 'ContentTypes';
+  public static RoutingName: string = 'ContentTypes';
```
Comment (natali-abyss) [inline demo/src/components/content-types/content-types.component.ts]: not used, remove

File: demo/src/components/credentials/credentials.component.ts [hunk 2]
```diff
@@ -35,24 +36,25 @@ import { IContentfulConfig, Ng2ContentfulConfig } from '../../../../src/ng2-cont
   `
 })
 export class CredentialsComponent implements OnInit {
-  static RoutingName = 'Credentials';
-  private model: IContentfulConfig = {
+  public static RoutingName: string = 'Credentials';
+
```
Comment (natali-abyss) [inline demo/src/components/credentials/credentials.component.ts]: not used

**Comment by mdmitriy** [comment]
used in "demo/src/app.tools.ts" line 19:
this.router.navigate([CredentialsComponent.RoutingName]);

File: demo/src/components/credentials/credentials.component.ts (hunk 2 above)
Comment (mdmitriy) [inline demo/src/components/credentials/credentials.component.ts]: used in "demo/src/app.tools.ts" line 19:
this.router.navigate([CredentialsComponent.RoutingName]);

File: package.json [hunk 3]
```diff
@@ -61,10 +63,15 @@
     "ts-loader": "0.8.2",
     "tslint": "3.15.1",
     "typescript": "1.8.10",
+    "typings": "^1.3.3",
```
Comment (natali-abyss) [inline package.json]: `^` - remove, only current version

**Pull request merged**